### PR TITLE
Replace the duration with month(int) for the data retention

### PR DESCRIPTION
**Pull request by yanmxa**
ref: https://issues.redhat.com/browse/ACM-7168

### CODE REVIEW

File: pkg/utils/time_test.go [hunk 1]
```diff
@@ -67,3 +68,35 @@ func TestParseDuration(t *testing.T) {
 		})
 	}
 }
+
+func TestParseRetention(t *testing.T) {
+	s := "-1y4m"
```
Comment (clyang82) [inline pkg/utils/time_test.go:73]: should be parsed error

Comment (yanmxa) [reply pkg/utils/time_test.go:73]: Done.

**Pull request merged**